packages/playwright-config-commons-ui/index.js

```diff
@@ -0,0 +1,35 @@
+const { devices } = require("@playwright/test");
+
+const config = {
+  timeout: 30 * 1000,
+  testMatch: /.*\.spec.js/,
+  reporter: "html",
+  use: {
+    actionTimeout: 0,
+    headless: false,
+    viewport: { width: 1280, height: 720 },
+    trace: "on-first-retry",
+  },
+  projects: [
```

Comment:
```suggestion
  webServer: {
    command: "pnpm dev",
    port: 3000,
    timeout: 120 * 1000,
    reuseExistingServer: !process.env.CI,
  },
  projects: [
```

Need webserver: https://nextjs.org/docs/testing#running-your-playwright-tests